lib/builder.js

```diff
@@ -296,6 +306,14 @@ function getPluginName(tree) {
 
 exports.getDescription = getDescription
 function getDescription (tree) {
+  if (typeof tree !== 'string') {
+    try {
```

Comment:
What is this protecting us against? I don’t think we want to swallow any/all errors here.

I think the swap to broccoli-node-info should address the originally reported issue properly without the try / catch...

Reply:
ya, sounds good, lets check for the specific error and only swallow that one

Reply:
🆗

Reply:
Done